### DISCUSSION TITLE
More EventToCommand Arg Converters - TextChangedEventArgs

### DISCUSSION THREAD

**Discussion by steveski** [CommunityToolkit/Maui#652] [New Feature Discussions]
I'm working on code to do Auto Complete on an Entry control and want to do data binding to my view model and I couldn't find an existing Event Arg Converter that suited. So I've added TextChangedEventArgsConverter to my project which works nicely. I'd like to pull request it to the CommunityToolkit.Maui package.

Any objections / suggestions?
If the suggestion is approved should I bring in others?

On another note, I can't actually build the solution from the latest main branch, or actually any point in history. I keep getting
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0016	Could not write to output file 'M:\Source\Source_2021-11-26\3rdPartyRepos\CommunityToolkit.Maui\src\CommunityToolkit.Maui.UnitTests\obj\GF\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\InternalTextStyleViewTextColorTo.g.shared.cs' -- 'Could not find a part of the path 'M:\Source\Source_2021-11-26\3rdPartyRepos\CommunityToolkit.Maui\src\CommunityToolkit.Maui.UnitTests\obj\GF\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.Generators.TextColorToGenerator\InternalTextStyleViewTextColorTo.g.shared.cs'.'	CommunityToolkit.Maui.UnitTests	M:\Source\Source_2021-11-26\3rdPartyRepos\CommunityToolkit.Maui\src\CommunityToolkit.Maui.UnitTests\CSC	1	Active
```

**Comment by TheCodeTraveler** [comment]
It looks like you're hitting the MAX_PATH issue on Windows. 

Here's more information to fix it: https://github.com/CommunityToolkit/Maui/issues/340#issuecomment-1081052363

**Reply by steveski** [reply]
Ah ok, I know that one. I wish after 37 years it could finally go away without needing a patch, or at least report the issue more meaningfully.
I ended up forking the project though and cloned to the same location, didn't have the issue building main on my fork

Thanks for the info

**Comment by CliffAgius** [comment]
Closed and answered
